site/docs/v1/tech/apis/applications.adoc

```diff
@@ -162,6 +163,125 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Search for Applications
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Applications and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
```

Comment:
```suggestion
[.api-authentication]
link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications by name
[.endpoint]
.URI
--
[method]#GET# [uri]#/api/application/search``?name=\{name\}``#
--

[.api-authentication]
link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications by tenant
[.endpoint]
.URI
--
[method]#GET# [uri]#/api/application/search``?tenantId=\{tenantId\}``#
--

```

These seem like the most common two options, so it is worth making them explicit.

Reply:
👍 Should we also illustrate sorting and paging on one of these? If so, do we provide a concrete example, like `orderBy=name`, or keep it generic like `orderBy={field}`?

Reply:
you are already (below) illustrating sorting. I don't think we need to illustrate paging, we do that here: https://fusionauth.io/docs/v1/tech/core-concepts/search#pagination

Though you could change the verbiage from 'users and entities' to 'search results'. (And also add a section here that talks about searches for other things, but that would be a nice to have.)

Reply:
@mmanes sorry if I was unclear. I thought we could have three sample urls

* one with all the parameters (which you had before my comment)
* one with application name
* one with tenant